### PR TITLE
fix: removed btc status unbonding from error return string

### PR DESCRIPTION
**Pull request by RafilxTenfen**
Closes: #139

### CODE REVIEW

File: x/btcstaking/types/btc_delegation.go [hunk 1]
```diff
@@ -30,7 +30,7 @@ func NewBTCDelegationStatusFromString(statusStr string) (BTCDelegationStatus, er
 	case "any":
 		return BTCDelegationStatus_ANY, nil
 	default:
-		return -1, fmt.Errorf("invalid status string; should be one of {pending, active, unbonding, unbonded, any}")
+		return -1, fmt.Errorf("invalid status string; should be one of {pending, active, unbonded, any}")
```
Comment (KonradStaniec) [inline x/btcstaking/types/btc_delegation.go:33]: oh can we also handle `verfified` in that function?

File: CHANGELOG.md [hunk 2]
```diff
@@ -50,6 +50,8 @@ messages inside `authz.MsgExec`
 
 * [#113](https://github.com/babylonlabs-io/babylon/pull/113) Add multibuild binary
 for upgrade handler `testnet` and `mainnet`.
+* [#140](https://github.com/babylonlabs-io/babylon/pull/140) Removed `unbonding`
```
Comment (KonradStaniec) [inline CHANGELOG.md:53]: this should probably be a bug fix

**Review by KonradStaniec — COMMENTED** [review]
about backports, if this not blocks our devnet1, I would release it in next minor release to avoid backporting hussle

File: x/btcstaking/types/btc_delegation.go (hunk 1 above)
Comment (RafilxTenfen) [reply x/btcstaking/types/btc_delegation.go]: done at https://github.com/babylonlabs-io/babylon/pull/140/commits/5b9cc82e2b433e6ccf0297bb01ec1613bfe397fc

File: CHANGELOG.md (hunk 2 above)
Comment (RafilxTenfen) [reply CHANGELOG.md]: moved at https://github.com/babylonlabs-io/babylon/pull/140/commits/d851cb1f0b5acaccea7182a9acf68f942568376b

**Review by KonradStaniec — APPROVED** [review]

**Pull request merged**